src/assets/text/support.ts

```diff
@@ -0,0 +1,16 @@
+export const support = {
+  main: [
+    `Found a bug? Can't connect your wallet? Need support?`,
```

Comment:
Use `"`, not back ticks. Weird that prettier is not fixing this automatically

Reply:
I think prettier will fix " vs ' but won't get in the way of ` since that behaves differently